qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -106,14 +106,44 @@ unmap_device()
     local dev=$1
     local pid=$2
 
-    _sudo rbd-nbd unmap ${dev}
-    rbd-nbd list-mapped | expect_false grep "^${pid}\\b" || return 1
+    _sudo rbd device --device-type nbd unmap ${dev}
+    rbd device --device-type nbd list | expect_false grep "^${pid}\\b" || return 1
     ps -C rbd-nbd | expect_false grep "^ *${pid}\\b" || return 1
 
     # workaround possible race between unmap and following map
     sleep 0.5
 }
 
+function test_default_pool()
+{
+    local pool=new_default_pool
+    local img=foo
+
+    ceph osd pool delete ${pool} ${pool} --yes-i-really-really-mean-it || true
+    ceph osd pool create ${pool} 32
+    rbd pool init ${pool}
+    # change default pool
+    rbd config global set global rbd_default_pool ${pool}
+    rbd config global get global rbd_default_pool | grep "^${pool}\\b"
+    rbd rm ${img} || true
+    rbd create --size 10M ${img}
+    # map/unmap test
+    dev=`_sudo rbd device --device-type nbd map ${img}`
+    pid=$(rbd device --device-type nbd list | awk -v pool=${pool} -v img=${img} -v dev=${dev} \
+        '$2 == pool && $3 == img && $5 == dev {print $1}')
+    test -n "${pid}"
+    _sudo rbd device --device-type nbd unmap ${dev}
+    rbd device --device-type nbd list | expect_false grep "^${pid}\\b" || return 1
+    ps -C rbd-nbd | expect_false grep "^ *${pid}\\b" || return 1
+    expect_false _sudo rbd device --device-type nbd map ${IMAGE}
```

Comment:
Ditto for unmap_device().  Try to extend existing helpers instead of open-coding.